src/nuxeo-cloud/day-to-day.md

```diff
@@ -0,0 +1,43 @@
+---
+title: 'Nuxeo Cloud Day-to-day'
+review:
+    comment: ''
+    date: '03-19-2020'
+    status: ok
+labels:
+    - nuxeo-cloud
+tree_item_index: 300
+---
+
+<div class="row" data-equalizer data-equalize-on="medium">
+<div class="column medium-4">
+{{#> panel type='secondary' match_height='true'}}
+### Deployment
+
+The Nuxeo Cloud Operations (NCO) team will deploy all changes and updates to the pre-prod and prod environments.
```

Comment:
change to "The Nuxeo Cloud team deploys all changes to Production and Pre-Production Environments."